src/machinery/eventListener.ts

```diff
@@ -22,18 +26,20 @@ const elementSelector = (selectedElement) => {
 let navigation = new Navigation([], elementSelector);
 
 loadedComponentStore.subscribe((value) => {
-  if (value.elements.length > 0) {
```

Comment:
Why did you remove it? If the elements are empty, it shouldn't create a navigation and attach keydown listener to it.

Reply:
Because we need to initialize this to set/unset selection and navigation on view change. So even though elements are an empty array, we need to know that in order to do right thing in for instance "Enter" case.